assets/js/components/dashboard-splash/dashboard-splash-service.js

```diff
@@ -36,15 +37,21 @@ class DashboardSplashService extends Component {
 		const { image, title, content, link, linkText, opposite } = this.props;
 
 		return (
-			<div className={ `googlesitekit-splash-service ${ opposite ? 'googlesitekit-splash-service--opposite' : '' }` } style={ { backgroundImage: `url(${ imagePath }service-${ image }.jpg)` } }>
+			<div
+				className={ classnames(
+					'googlesitekit-splash-service',
+					{ 'googlesitekit-splash-service--opposite': opposite }
+				) }
+				style={ { backgroundImage: `url(${ imagePath }service-${ image }.jpg)` } }
+			>
 				<div className="mdc-layout-grid">
 					<div className="mdc-layout-grid__inner">
-						<div className={ `
-							mdc-layout-grid__cell
-							mdc-layout-grid__cell--span-5-tablet
-							mdc-layout-grid__cell--span-7-desktop
-							${ opposite ? 'mdc-layout-grid__cell--start-3-tablet mdc-layout-grid__cell--start-5-desktop mdc-layout-grid__cell--offset-5-tablet' : '' }
-						` }>
+						<div className={ classnames(
+							'mdc-layout-grid__cell',
+							'mdc-layout-grid__cell--span-5-tablet',
+							'mdc-layout-grid__cell--span-7-desktop',
+							{ 'mdc-layout-grid__cell--start-3-tablet mdc-layout-grid__cell--start-5-desktop mdc-layout-grid__cell--offset-5-tablet': opposite }
```

Comment:
As mentioned in https://github.com/google/site-kit-wp/pull/1105/files#r377545433:

It would be cleaner to read this with each `className` as its own line. I know it's a bit "redundant" but it'll great improve readability, eg:

```suggestion
							{
								'mdc-layout-grid__cell--start-3-tablet': opposite.
								'mdc-layout-grid__cell--start-5-desktop': opposite,
								'mdc-layout-grid__cell--offset-5-tablet': opposite,
							}
```